src/Evolution/DgSubcell/Tags/Mesh.hpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+
+/// \cond
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace evolution::dg::subcell::Tags {
+/// The mesh on the subcells
+template <size_t VolumeDim>
+struct Mesh : db::SimpleTag {
+  using type = ::Mesh<VolumeDim>;
```

Comment:
I don't really see a need for that with these tags. I don't think they're ever retrieved by name, and if the active grid is FD/FV then it's not clear what prefix (if any) should be used.

Reply:
This is no hill I want to die on, though when printing DataBox contents and it just says "Mesh" twice I would find that confusing. Since this is in a `dg::subcell` namespace I feel like `SubcellMesh` would describe this well...